tests/unit/EntityInfo.spec.ts

```diff
@@ -366,13 +366,12 @@ describe('EntityInfo - Click Tests - Dissolutions', () => {
 
   it('displays the Dissolve this Company button', async () => {
     // mount the component and wait for everything to stabilize
-    const wrapper = shallowMount(EntityInfo, {
+    const wrapper = mount(EntityInfo, {
```

Comment:
This shouldn't be needed since EntityInfo doesn't need any sub-component to be rendered (stubs should be enough). What error does this fix? A Vuetify component maybe?

Reply:
since I moved, the button inside the tooltip, shallowMount will not render the button inside vuetify, Else test case will fail, button will not exist

Reply:
Ah, makes sense.